src/stdatamodels/model_base.py

```diff
@@ -349,50 +350,27 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
 
-    def _drop_arrays(self):
-        def _drop_array(d):
-            # Walk tree and delete numpy arrays
-            if isinstance(d, dict):
-                for val in d.values():
-                    _drop_array(val)
-            elif isinstance(d, list):
-                for val in d:
-                    _drop_array(val)
-            elif isinstance(d, np.ndarray):
-                del d
```

Comment:
This method had no effect because `del d` just deletes the local variable, it doesn't modify the dict or list that contained the array.